README.md

```diff
@@ -64,6 +64,8 @@ description = "My content"
 website = "https://mywebsite.com"
 // If enabled, attempt to sign .js JavaScript files. Disabled by default
 signJavaScript = true
+// If unspecified, both sha1 and sha256 signatures will be appended. Will be passed to signtool.exe as the /fd option.
+hash = "sha256" 
```

Comment:
This should probably be an array so that folks can ask for multiple hashes if they want

Reply:
I'll do that and expose the enums, too.

BTW, @MarshallOfSound , thanks for everything you've done with electron. Mountain of work. Kudos.

Reply:
@MarshallOfSound that should do it, I think? Holler if I misunderstood.

Reply:
FWIW it's a bit less foot-gun-esque if you only supported [sha1, sha256] or [sha256]:  signing only with sha1 will not work with any currently supported version of Windows (according to Microsoft's current documentation -- I don't have a windows 7 test rig to test this assertion on)